share/spice/news/news.js

```diff
@@ -93,10 +93,27 @@
                 meta: {
                     count: goodStories.length,
                     searchTerm: searchTerm,
-                    itemType: 'News articles'
+                    itemType: 'News articles',
+                    rerender: [
+                        'image'
+                    ]
                 },
                 templates: {
                     item: 'news_item'
```

Comment:
Oh, that's interesting, maybe. I'll defer to @chrismorast and @abeyang, maybe there's a way to consolidate the two.